### PR TITLE
Improvements to SLA page per Tinas doc comments

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

File: docs/eigenda/operator-guides/protocol-SLA/protocol-SLA.md [hunk 1]
```diff
@@ -4,73 +4,51 @@ title: Protocol SLA
...
-  data it signed until all the data is expired.
+  - The operator will continue to be responsible for storing and serving the data it signed until all the data is expired.
 - **Free:** The operator becomes free of responsibilities starting block `D+1`.
 
-Note: if the operator re-opts in the quorum at any point from `B` to `D`, the above
-lifecycle will be restarted.
+Note: if the operator re-opts in the quorum at any point from `B` to `D`, the above lifecycle will be restarted.
 
-## Accountability
+## Accountability Measures, Policies, and Actions
```
Comment (jianoaix) [inline docs/eigenda/operator-guides/protocol-SLA/protocol-SLA.md:51]: Measures -> Measurements?

File: docs/eigenda/operator-guides/protocol-SLA/protocol-SLA.md [hunk 2]
```diff
@@ -4,73 +4,51 @@ title: Protocol SLA
...
 
 ### Liveness
```
Comment (jianoaix) [inline docs/eigenda/operator-guides/protocol-SLA/protocol-SLA.md:54]: Inside the table below, is there a way to break the line between Soft measure and Hard measure?


Screenshot 2024-04-08 at 12 31 36 PM

**Review by jianoaix — COMMENTED** [review]
lgtm

File: docs/eigenda/operator-guides/protocol-SLA/protocol-SLA.md (hunk 2 above)
Comment (wesfloyd) [reply docs/eigenda/operator-guides/protocol-SLA/protocol-SLA.md:54]: Yes - stackoverflow says <br> will fix it. Updated the table to include a BR, please see the revision and let me know if you have concerns.

**Review by jianoaix — APPROVED** [review]

**Review by tinahaibodi — APPROVED** [review]

**Pull request merged**